cfgov/unprocessed/css/blog-docs.less

```diff
@@ -420,13 +420,19 @@
     }
 }
 
-.filtered-by {
+.filtered-by, .error-container {
     .cf-notification  {
         display: block;
         margin: @gutter__em 0 @gutter__em * 2;
     }
 }
 
+.error-container {
+    .cf-notification  {
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/commit/606b7c1d369eb210b18ded6cb7f660ffbe3bf87e